### PR TITLE
Call new remove-curator 0chain endpoint

### PR DESCRIPTION
**Pull request by Sriep**
Requires https://github.com/0chain/0chain and https://github.com/0chain/gosdk/pull/201. 

go.mod and go.sum will need to be updated with gosdk commit when https://github.com/0chain/gosdk/pull/201 is merged.

### CODE REVIEW

**Review by cnlangzi — CHANGES_REQUESTED** [review]

File: cmd/removecurator.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		curatorID, err := flags.GetString("curator")
+		if err != nil {
+			log.Fatal("invalid 'curator_id' flag: ", err)
+		}
+
+		_, err = sdk.RemoveCurator(curatorID, allocationID)
+		if err != nil {
+			log.Fatal("Error adding curator:", err)
+		}
+		log.Print(curatorID + " removed " + curatorID + " as a curator to allocation " + allocationID)
```
[the remark this answers is not in the document]
Comment (Sriep) [reply cmd/removecurator.go:39]: Not sure, but I am just copying everyone else.

File: cmd/addcurator.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ var addCuratorCmd = &cobra.Command{
 		if err != nil {
 			log.Fatal("Error adding curator:", err)
 		}
-		log.Print(curatorID + " added as a curator to allocation " + allocationID)
+		log.Print(curatorID + " added " + curatorID + " as a curator to allocation " + allocationID)
```
[the remark this answers is not in the document]
Comment (Sriep) [reply cmd/addcurator.go:39]: Not sure, but I am just copying everyone else. zbox is a straightforward `cobra` command-line application so there is no obvious reason to anything other than "log".

[the remark this answers is not in the document]
Comment (cnlangzi) [reply cmd/addcurator.go:39]: if so, please remove log.Print for debugging. It break `--silent` feature

File: cmd/removecurator.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (cnlangzi) [reply cmd/removecurator.go:39]: if so, please remove log.Print for debugging. It break `--silent` feature

File: cmd/addcurator.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (Sriep) [reply cmd/addcurator.go:39]: I don't understand.
```
"silent", false, "Do not show interactive sdk logs (shown by default)"
```
The output from `log` is not part of the sdk logs.

File: cmd/removecurator.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sriep) [reply cmd/removecurator.go:39]: I don't understand.
```
"silent", false, "Do not show interactive sdk logs (shown by default)"
```
The output from `log` is not part of the sdk logs.

**Review by cnlangzi — APPROVED** [review]
LGTM

**Pull request merged**